main/settings_course_etl.py

```diff
@@ -13,6 +13,11 @@
 EDX_LEARNING_COURSE_BUCKET_PREFIX = get_string(
     "EDX_LEARNING_COURSE_BUCKET_PREFIX", "simeon-mitx-course-tarballs"
 )
+# EdX API is rate limited. This setting can be used to read from a local datafile
+# isntead of the API. DEVELOPMENT ONLY.
+EDX_LOCAL_API_DATAFILE = get_string("EDX_LOCAL_API_DATAFILE", None)
```

Comment:
@mbertrand Is there a convenient way to give an optional argument to `pipelines.oll_etl`? Maybe it would be better to have this be a CLI argument rather than a setting.